library/src/main/java/com/novoda/downloadmanager/lib/DownloadBatch.java

```diff
@@ -50,6 +50,35 @@ public boolean isDeleted() {
         return this == DELETED;
     }
 
+    /**
+     * Return time when this download will be ready for its next action, in
```

Comment:
I'm not 100% sure, the main service loops sets alarm manager wake ups based on this retry time amount